h2integrate/core/model_base.py

```diff
@@ -0,0 +1,23 @@
+import openmdao.api as om
+
+
+class CostModelBaseClass(om.ExplicitComponent):
+    def initialize(self):
```

Comment:
Please add a docstring explaining that this baseclass should be used for any cost model; maybe briefly explain that the built-in outputs are used by the finance model. It might not be intuitive to some users why they need to provide a `cost_year`, especially if they're not thinking about the financial modeling.